src/main/clojure/clara/tools/testing_utils.cljc

```diff
@@ -0,0 +1,89 @@
+#?(:clj
+   (ns clara.tools.testing-utils
+     "Internal utilities for testing clara-rules and derivative projects.  These should be considered experimental
+  right now from the perspective of consumers of clara-rules, although it is possible that this namespace 
+  will be made part of the public API once its functionality has proven robust and reliable.  The focus, however,
+  is functionality needed to test the rules engine itself."
+     (:require [clara.rules.update-cache.core :as uc]
+               [clara.rules.update-cache.cancelling :as ca]
```

Comment:
This should be the only `:clj` only ns from the ones I see here.
This relates to https://github.com/cerner/clara-rules/pull/334/files#r133448443

So the whole `ns` header should be
```clj
(ns clara.tools.testing-utils
   "Internal utilities for testing clara-rules and derivative projects.  These should be considered experimental
  right now from the perspective of consumers of clara-rules, although it is possible that this namespace 
  will be made part of the public API once its functionality has proven robust and reliable.  The focus, however,
  is functionality needed to test the rules engine itself."
   (:require [clara.rules.update-cache.core :as uc]
             #?(:clj [clara.rules.update-cache.cancelling :as ca])
             [clara.rules.compiler :as com]
             [clara.macros :as m]
             [clara.rules.dsl :as dsl]))
```